docs/network-flow-visibility.md

```diff
@@ -202,30 +203,70 @@ helm install theia build/charts/theia -n flow-visibility --create-namespace \
   --set=clickhouse.storageSize="2Gi"
 ```
 
-The ClickHouse TCP service is used by the Flow Aggregator. If you have changed the TCP port,
-please update the `databaseURL` following
-[Flow Aggregator Configuration](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#configuration-1).
+The ClickHouse TCP Service is used by the Flow Aggregator. If you have changed
+the TCP port, please update the `clickHouse.databaseURL` in the Flow Aggregator
+Helm Chart values.
+
+The ClickHouse credentials are used in the Flow Aggregator. If you have changed
+the ClickHouse credentials, please update the `clickHouse.connectionSecret` in the
+Flow Aggregator Helm Chart values.
+
+Please refer to the Flow Aggregator Helm Chart
+[README](https://github.com/antrea-io/antrea/blob/main/build/charts/flow-aggregator/README.md)
+and [Flow Aggregator Configuration](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#configuration-1)
+to learn more about ClickHouse configuration options in Flow Aggregator.
+
+##### ClickHouse Cluster
+
+In v0.1, Theia deploys ClickHouse in a non-cluster mode. Starting with v0.2,
+Theia supports ClickHouse database cluster with data replication, and deprecates
+the non-cluster mode. You can upgrade from v0.1 to v0.2 without data losing if
+you use PersistentVolume for ClickHouse. To downgrade from v0.2 to v0.1, you
+need to uninstall ClickHouse, delete related data if you use PersistenVolume
+and redeploy it.
+
+A ClickHouse cluster consists of one or more shards. Shards refer to the servers
+that contain different parts of the data. You can deploy multiple shards to scale
+the cluster horizontally. Each shard consists of one or more replica hosts.
+Replicas indicates storing the same data on multiple Pods to improve data
```

Comment:
`indicates` -> `indicate`

Reply:
This is introduced in the ClickHouse cluster PR. Updated there and rebased. Thanks!